src/Sepes.Infrastructure/Service/DataModelService/SandboxModelService.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Sepes.Infrastructure.Constants;
+using Sepes.Infrastructure.Exceptions;
+using Sepes.Infrastructure.Model;
+using Sepes.Infrastructure.Model.Context;
+using Sepes.Infrastructure.Service.DataModelService.Interface;
+using Sepes.Infrastructure.Service.Interface;
+using Sepes.Infrastructure.Service.Queries;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Sepes.Infrastructure.Service.DataModelService
+{
+    public class SandboxModelService : ModelServiceBase<Sandbox>, ISandboxModelService
+    {
+        public SandboxModelService(IConfiguration configuration, SepesDbContext db, ILogger<SandboxModelService> logger, IUserService userService)
+            : base(configuration, db, logger, userService)
+        {
+
+        }
+
+        public async Task<Sandbox> GetByIdWithoutPermissionCheckAsync(int sandboxId)
+        {
+            return await SandboxBaseQueries.AllSandboxesBaseQueryable(_db).SingleOrDefaultAsync(sb => sb.Id == sandboxId);
+        }
+
+        public async Task<Sandbox> GetByIdAsync(int id, UserOperation userOperation, bool withIncludes = false, bool disableTracking = false)
+        {
+            var sandboxQueryable =
+               (withIncludes ? SandboxBaseQueries.ActiveSandboxesWithIncludesQueryable(_db) : SandboxBaseQueries.ActiveSandboxesMinimalIncludesQueryable(_db));
```

Comment:
Is this an SQL statement?

Reply:
Not in its self, but inside SandboxBaseQueries lies linq queries related to sandbox